src/components/application_manager/src/hmi_state.cc

```diff
@@ -73,8 +86,10 @@ DEPRECATED HmiState::HmiState(uint32_t app_id,
     , app_mngr_(app_mngr)
     , hmi_level_(mobile_apis::HMILevel::INVALID_ENUM)
     , audio_streaming_state_(mobile_apis::AudioStreamingState::INVALID_ENUM)
+    , video_streaming_state_(mobile_apis::VideoStreamingState::INVALID_ENUM)
```

Comment:
This modifies a deprecated method. The next version of SDL Core is 5.0 a major version change. The DEPRECATED tag can be removed

Reply:
@JackLivio , agree.
see 3cec3966343d95fff2626ce11a46aedd42d0eea8